src/libraries/System.Text.Json/tests/Serialization/ContinuationTests.cs

```diff
@@ -32,6 +32,12 @@ private static IEnumerable<bool> IgnoreNullValues
         [MemberData(nameof(TestData))]
         public static async Task ContinuationShouldWorkAtAnyPosition_Class_Class(int paddingLength, bool ignoreNullValues)
         {
+            if (PlatformDetection.IsMonoRuntime && PlatformDetection.IsWindows)
```

Comment:
Elsewhere I see us do:
```C#
[ActiveIssue("https://github.com/dotnet/runtime/issues/42677", platforms: TestPlatforms.Windows, runtimes: TestRuntimes.Mono)]
```
or
```C#
[ActiveIssue("https://github.com/dotnet/runtime/issues/42677", TestPlatforms.Windows, TargetFrameworkMonikers.Netcoreapp, TestRuntimes.Mono)]
```

Reply:
Thanks, updated to use this pattern instead.